Tests/AK/TestFixedArray.cpp

```diff
@@ -63,10 +63,16 @@ TEST_CASE(no_allocation)
 
     EXPECT_NO_CRASH("Move", [&] {
         FixedArray<int> moved_from_array = FixedArray<int>::must_create_but_fixme_should_propagate_errors(6);
-        NoAllocationGuard guard;
+
+        // We need to do this because calling a destructor manually would be UB
+        // otherwise, and the guard is no extractable in to its own scope separate
```

Comment:
Typos:
`no extractable` -> `not extractable`
`in to its own` -> `into its own`